sepolia/2024-06-04-initialize-fault-proof-proxies/Makefile

```
@@ -0,0 +1,26 @@
+include ../../Makefile
+include ../.env
+include .env
+
+ifndef LEDGER_ACCOUNT
+override LEDGER_ACCOUNT = 0
```

Most signers are using their second ledger address for testnet
```suggestion
override LEDGER_ACCOUNT = 1
```